skfp/datasets/tdc/tox/ames.py

```diff
@@ -22,6 +22,18 @@ def load_ames(
     verbose: bool = False,
 ) -> Union[pd.DataFrame, tuple[list[str]], np.ndarray]:
     """
+    Load and return the AMES dataset from TDC benchmark.
+
+    The task is to predict the ability of drugs to cause
+    genetic mutations given a set of molecules [1]_.
```

Comment:
1. Maybe not "ability", since it's an unwanted effect
2. Also cite TDC
3. Expand the description a bit, TDC has a nice one